### PR TITLE
DEP Update jquery-ui in thirdparty folder

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1423

jquery-ui v1.9.2  -> v1.13.2

This should have been upgraded as part of 1.12, and since there's an existing regression I think we should do a patch release in 1.12 rather than waiting for 1.13

Updated because of this answer https://stackoverflow.com/a/37925405 - javascript error matched what console error I saw after running `yarn dev` and replicating issue

Issue could be replicated by 
- Creating a test member in the security section of the cms
- Editing a group in the group section of the security section and attempting to add the new member to the group using the autocompleter

### CODE REVIEW

File: client/src/legacy/LeftAndMain.js [hunk 1]
```diff
@@ -889,12 +889,12 @@ $.entwine('ss', function($) {
...
         var id = $(el).attr('id');
         if(!id) return; // we need a unique reference
-        if(!$(el).data('tabs')) return; // don't act on uninit'ed controls
+        if(!$(el).data('uiTabs')) return; // don't act on uninit'ed controls
 
         // Allow opt-out via data element or entwine property.
         if($(el).data('ignoreTabState') || $(el).getIgnoreTabState()) return;
 
-        selectedTabs.push({id:id, selected:$(el).tabs('option', 'selected')});
+        selectedTabs.push({id:id, selected:$(el).tabs('option', 'active')});
```
Comment (emteknetnz) [inline client/src/legacy/LeftAndMain.js:897]: 'selected' was renamed to 'active' - https://bugs.jqueryui.com/ticket/7135

File: client/src/legacy/LeftAndMain.js [hunk 2]
```diff
@@ -889,12 +889,12 @@ $.entwine('ss', function($) {
       this.find('.cms-tabset,.ss-tabset').each(function(i, el) {
         var id = $(el).attr('id');
         if(!id) return; // we need a unique reference
-        if(!$(el).data('tabs')) return; // don't act on uninit'ed controls
+        if(!$(el).data('uiTabs')) return; // don't act on uninit'ed controls
```
Comment (emteknetnz) [inline client/src/legacy/LeftAndMain.js:892]: 'tabs' was renamed to 'uiTabs' in jquery-ui 1.10 - I couldn't find a reference of where this was done, though console.log'ing `$el.data()` showed it to now be `'uiTabs'`

File: client/src/styles/legacy/_uitheme.scss [hunk 3]
```diff
@@ -16,10 +16,15 @@
...
 .ui-widget-content,
-.ui-widget {
+.ui-widget,
+.ui-widget.ui-widget-content {
   color: $body-color;
   font-size: 1em;
   border: 0;
+
+  a {
+    color: $link-color;
```
Comment (emteknetnz) [inline client/src/styles/legacy/_uitheme.scss:26]: These CSS changes are effectively to revert some CSS additions that went into later versions of jquery-ui

**Comment by GuySartorelli** [comment]
@emteknetnz In the installer CI run you linked to, _everything_ is failing... does that link need to be updated to a more recent run? Or does this PR legitimately break everything?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
The cursor should be `pointer` when hovering over tabs.

The icon alignment (super minor so not too fussed) and text colour of expandable composite fields is not correct.
**before:**
image
**this PR:**
image

The arrow icon on expandable composite fields disappears when expanding the field - it should not.

Everything else looks fine - and the reported regression is resolved.

**Comment by emteknetnz** [comment]
@GuySartorelli updated

**Comment by GuySartorelli** [comment]
@emteknetnz Can you please respond to my question about the installer CI run?
You also only seem to have addressed one of the three regressions I mentioned.

**Review by GuySartorelli — APPROVED** [review]
Looks good, works great locally.

**Comment by GuySartorelli** [comment]
Released as [1.12.1](https://github.com/silverstripe/silverstripe-admin/releases/tag/1.12.1)

**Pull request merged**